README.md

```diff
@@ -1,2 +1,22 @@
-# final-ci-docker
-Docker files for final-ci service.
+# Dockerized final-api application
+
+This solution allows running Dockerized version of [Final-API](https://github.com/AVGTechnologies/final-api).
+
+## What does it mean "dockerised"? What is the motivation?
```

Comment:
Hey, is it dockerized or dockerised? you should choose one 'version' and use it everywhere :)